.github/workflows/unit-test.yaml

```diff
@@ -10,17 +10,16 @@ jobs:
     strategy:
       matrix:
         go: [1.16]
-        java: [11, 17]
-        postgres: [9.6, 12]
-        suite: [test, integration]
+        java: [11,17]
+        postgres: [12]
```

Comment:
as @silvestre mentioned, internally we rely on 9.6 atm, so does the bosh release.  We shouldn't remove that yet.

Reply:
I think what we consider doing is to remove postgress altogther and move any failing tests into the `integration suite` this would highlight the dependencies.

Or just run the db test suite with all the databases and just use postgres 12 (since we are moving to that shortly) in the rest.

Consider making it into a db backard and forward compatibility test